code/modules/projectiles/guns/throw/crossbow.dm

```diff
@@ -29,41 +21,48 @@
 	if(cell && severity)
 		emp_act(severity)
 
-/obj/item/gun/throw/crossbow/update_icon()
-	if(!tension)
-		if(!to_launch)
-			icon_state = "[initial(icon_state)]"
-		else
-			icon_state = "[initial(icon_state)]-nocked"
-	else
-		icon_state = "[initial(icon_state)]-drawn"
+obj/item/gun/throw/crossbow/update_icon()
+	icon_state = "[initial(icon_state)]"
+	if(cell)
+		icon_state += "-Cell"
+	if(to_launch)
+		icon_state += "-Loaded"
+	if(tension)
+		icon_state += "-Charged"
+	if(cell.charge >= 500 && tension)
+		icon_state +="-Hot"
 
 /obj/item/gun/throw/crossbow/examine(mob/user)
 	. = ..()
 	if(cell)
 		. += "<span class='notice'>\A [cell] is mounted onto [src]. Battery cell charge: [cell.charge]/[cell.maxcharge]"
+		update_icon()
 	else
 		. += "<span class='notice'>It has an empty mount for a battery cell.</span>"
 
-/obj/item/gun/throw/crossbow/modify_projectile(obj/item/I, on_chamber = 0)
-	if(cell && on_chamber && istype(I, /obj/item/arrow/rod))
+/obj/item/gun/throw/crossbow/modify_projectile(obj/I,)
+	if( tension && (cell.charge >= 1999))
```

Comment:
Remove this space between the `(` and `cell.charge`

Also, you should change `1999` to `2000` because in the event of the cell having exactly 1999 charge, it won't use any (given you are doing `cell.use(2000)` later on)

I would put all your code below this in something like the following:

```
if(tension && cell.use(2000))
    code_goes_here
```
This is because the `use()` proc returns `TRUE` if the cell actually used the charge (and didn't fail), so we know we can proceed with the rest of the code

Reply:
its not exactly 1999 as you can see the cell charge thing says >= that means more than 1999
starting from 2000

Reply:
`x >= 1999` means: x is greater than or equal to 1999
If you want starting from 2000, it should probably be as Citnited suggested.

`x > 1999` also works for integers, but if `cell.charge` was a floating value, then `1999.5` would pass the check. It seems you want strict 2000+ ?